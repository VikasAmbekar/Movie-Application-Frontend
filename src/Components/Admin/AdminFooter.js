import React from "react";
import classes from "./AdminFooter.module.css";
// Import only MDBFooter to potentially reduce bundle size
import { MDBFooter } from "mdb-react-ui-kit";

const AdminFooter = () => {
  return (
    <>
      <MDBFooter
        className={`${classes.footerBackground} text-center text-white`}
        aria-label="footer"
      >
        <div className={`${classes.copyrightContainer} p-3`}>
          Â© 2023 Copyright
          <a
            className={`${classes.copyrightLink}`}
            href="#"
            aria-label="MovieMate link"
          >
            MovieMate
          </a>
        </div>
      </MDBFooter>
    </>
  );
};

export default AdminFooter;
