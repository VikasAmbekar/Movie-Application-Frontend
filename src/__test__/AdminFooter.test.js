import { shallow } from "enzyme";
import Enzyme from "enzyme";
import Adapter from "@cfaester/enzyme-adapter-react-18";
import AdminFooter from "../Components/Admin/AdminFooter";
Enzyme.configure({ adapter: new Adapter() });
jest.mock("react-router-dom");

describe("AdminFooter test case", () => {
  it("class test", () => {
    let wrapper = shallow(<AdminFooter />);
    expect(wrapper.exists(".footerDiv")).toEqual(true);
  });

  it("Testing tag elements", () => {
    let wrapper = shallow(<AdminFooter />);
    expect(wrapper.find("a").text()).toEqual("MovieMate");
  });
});
