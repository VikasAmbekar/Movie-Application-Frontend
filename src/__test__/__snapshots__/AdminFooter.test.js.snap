// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdminFooter test case renders correctly 1`] = `
<div>
  <footer
    aria-label="footer"
    class="footerBackground text-center text-white"
  >
    <div
      class="copyrightContainer p-3"
    >
      Â© 2023 Copyright
      <a
        aria-label="MovieMate link"
        class="copyrightLink"
        href="#"
      >
        MovieMate
      </a>
    </div>
  </footer>
</div>
`;
